### DISCUSSION TITLE
Welcome to mpp-solar Discussions!

### DISCUSSION THREAD

**Discussion by jblance** [jblance/mpp-solar#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 20after4** [comment]
Hello, my name is Mukunda and I've recently purchased a PIP-3048LV-MK. I discovered your project from some discussions on diysolarforum.com. I've already ran into a couple of small issues but was able to resolve them with very slight code changes. I have submitted pull requests in hopes that it may help others who also have a PIP-3048LV.

Thank you for creating this project and sharing it here! It seems like a really well designed piece of software and I look forward to helping in any way that I can to improve it.

**Reply by jblance** [reply]
Hi

Glad it can be of some use to you.

Thanks heaps for the PRs!!

I welcome any improvements or suggestions

Regards
John
